### PR TITLE
Cancel Booking request body

### PR DESCRIPTION
**Pull request by gimpster**
Return request body for Cancel Booking endpoint and CancellationRequest for Booking object. These were mistakenly removed when the cancellation endpoints were restructured.

### CODE REVIEW

File: docs/openapi_v1.0.0-alpha.yaml [hunk 1]
```diff
@@ -510,6 +512,73 @@ components:
...
+          format: date-time
+          description: |
+            This MUST be an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) compliant date and time. The value represents the time at which the cancellation request was originally started. This timestamp MUST be used for any validation against a cancellation policy. This is important because there may be some delay in confirming this cancellation request during the 2-phase workflow which could finish just after the cancellation policy cutoff has elapsed.
+          example: '2019-10-31T08:30:00Z'
+        utcHoldExpiration:
+          type: string
+          format: date-time
+          nullable: true
+          description: |
+            This MUST be an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) compliant date and time. The value represents the time at which the cancellation request hold will be released. This SHOULD be equivalent to the time calculated by adding `holdExpirationMinutes` to the current UTC time but MAY be either earlier or later than the requested duration.
```
Comment (dusanzahoransky) [inline docs/openapi_v1.0.0-alpha.yaml]: The cancellation is a one-step transaction now, isn't it? This field is not needed

File: docs/openapi_v1.0.0-alpha.yaml [hunk 2]
```diff
@@ -510,6 +512,73 @@ components:
...
+    CancellationRequest:
+      type: object
+      required:
+        - reason
+        - reasonDetails
+        - status
+        - refund
+        - utcRequestedAt
+        - utcHoldExpiration
+        - utcConfirmedAt
```
Comment (dusanzahoransky) [inline docs/openapi_v1.0.0-alpha.yaml]: utcConfirmedAt and utcResolvedAt should be just optional, I don't think they are essential for a cancellation request.

File: docs/openapi_v1.0.0-alpha.yaml (hunk 1 above)
Comment (ollym) [reply docs/openapi_v1.0.0-alpha.yaml]: Correct, this isn't needed anymore

File: docs/openapi_v1.0.0-alpha.yaml (hunk 2 above)
Comment (ollym) [reply docs/openapi_v1.0.0-alpha.yaml]: The idea was if you didnt support utcConfirmedAt / utcResolvedAt they can just be set to the datetime the cancellation happened

File: docs/openapi_v1.0.0-alpha.yaml (hunk 1 above)
Comment (gimpster) [reply docs/openapi_v1.0.0-alpha.yaml]: Ah, yup. Missed that one when I was bringing back the old models. I'll remove it.

**Comment by ollym** [comment]
Looks good now

File: docs/openapi_v1.0.0-alpha.yaml (hunk 2 above)
Comment (gimpster) [reply docs/openapi_v1.0.0-alpha.yaml]: Thinking about the workflow on these, back when cancellation was a 2-phase operation, these two fields were used to facilitate delayed cancellation confirmation (similar to the delayed booking confirmation that we support). I'm wondering if we still need to support that use case? If we do, then maybe this should be simplified to a single field like `utcConfirmedAt` as an optional field? If we don't need to support that use case, then I think this field would just always return the time that the cancellation was processed by the server.

**Pull request merged**